build/debs/kubelet-postinst.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -x /bin/systemctl ] && systemctl is-active systemd-resolved > /dev/null 2>&1 ; then
+  mkdir -p /etc/systemd/system/kubelet.service.d
+  cat <<EOF > /etc/systemd/system/kubelet.service.d/09-systemd-resolved.conf
+[Service]
+Environment="KUBELET_RESOLVER_ARGS=--resolv-conf=/run/systemd/resolve/resolv.conf"
+ExecStart=/usr/bin/kubelet $KUBELET_RESOLVER_ARGS
```

Comment:
I think you should omit this and only include the env argument

Reply:
I believe this is needed in the case that the kubelet is installed but kubeadm is not.

That said, I think this would also need to have a preeceeding line with `ExecStart=` to work in that case.

Reply:
Oh didn't realize this was the kubelet deb, not the kubeadm as I thought.
Yeah, the extra execstart is needed then

Reply:
Cool. Will sort out in a bit.

Following from https://github.com/kubernetes/kubernetes/pull/63887#issuecomment-389384302 should I also move the Kubelet drop in to being under `/usr/lib/system` ?

Reply:
@randomvariable I think it is best to hold off until we have a better plan around how we could manage the transition.